apps/server/lib/lexical/server/provider/handlers/code_action.ex

```diff
@@ -1,18 +1,42 @@
 defmodule Lexical.Server.Provider.Handlers.CodeAction do
   alias Lexical.Protocol.Requests
   alias Lexical.Protocol.Responses
-  alias Lexical.Server.Provider.CodeAction.ReplaceWithUnderscore
-  alias Lexical.Server.Provider.CodeAction.ReplaceWithUnderscore
+  alias Lexical.Protocol.Types
+  alias Lexical.Protocol.Types.CodeAction, as: CodeActionResult
+  alias Lexical.Protocol.Types.Workspace
```

Comment:
I've thought about this a few times, and I'm wondering if we should always start using the `Types` prefix when dealing with protocol types. You're already doing it below with `%Types.Diagnostic{}`, and I think just using `Types.CodeAction` is more clear than aliasing as `CodeActionResult`.

```suggestion
```